spec/factories.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user do
+    email { 'poczta@damianstasiak.pl' }
```

Comment:
The purpose of using factories is to make them reusable. To easly create X objects of the same model for instance.  

It will be impossible here, as user email address should be uniq. Please use `sequence` here.

Another recommendation is to use `example.com` domain in specs. It's special domain for that purpose. It makes you sure you will never hit real data/apis.